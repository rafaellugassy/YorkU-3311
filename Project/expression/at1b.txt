restart

modulo
analyze
simplify

restart

modulo
int_value (5)
int_value (2)
analyze
simplify 