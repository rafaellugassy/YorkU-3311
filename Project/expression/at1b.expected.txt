  Expression currently specified: ?
  Report: Expression is initialized.
->restart
  Expression currently specified: ?
  Report: Error (Initial expression cannot be reset).
->modulo
  Expression currently specified: (? mod nil)
  Report: OK.
->analyze
  Expression currently specified: (? mod nil)
  Report: Error (Expression is not yet fully specified).
->simplify
  Expression currently specified: (? mod nil)
  Report: Error (Expression is not yet fully specified).
->restart
  Expression currently specified: ?
  Report: OK.
->modulo
  Expression currently specified: (? mod nil)
  Report: OK.
->int_value(5)
  Expression currently specified: (5 mod ?)
  Report: OK.
->int_value(2)
  Expression currently specified: (5 mod 2)
  Report: OK.
->analyze
  Expression currently specified: (5 mod 2)
  Report: (5 mod 2) is type-correct.
->simplify
  Expression currently specified: (5 mod 2)
  Report: 1
